sdk/ai/azure-ai-generative/azure/ai/generative/evaluate/_metric_handler.py

```diff
@@ -3,7 +3,7 @@
 # ---------------------------------------------------------
 import copy
 
-from azure.ai.generative.evaluate._constants import TYPE_TO_KWARGS_MAPPING
+from azure.ai.generative.evaluate._constants import TYPE_TO_KWARGS_MAPPING, CHAT_RAG, QA_RAG
```

Comment:
Where do we use these 2 imported values?

Reply:
Thanks for catching that, will handle it in a subsequent PR